lib/service/deeplink_service.dart

```diff
@@ -229,7 +229,7 @@ class DeeplinkServiceImpl extends DeeplinkService {
     ];
     if (!_configurationService.isDoneOnboarding()) {
       memoryValues.deepLink.value = link;
-      await injector<AccountService>().restoreIfNeeded(isCreateNew: false);
+      await injector<AccountService>().restoreIfNeeded();
```

Comment:
isCreateNew: false
-> when it navigate to connect page, it will create addresses there